tests_smoke/test_admin_one_off.py

```diff
@@ -0,0 +1,52 @@
+import time
+
+import requests
+from common import Config, pretty_print  # type: ignore
+from notifications_python_client.authentication import create_jwt_token
+
+
+def test_admin_one_off(notification_type: str):
+    print(f"test_admin_one_off ({notification_type})... ", end="", flush=True)
+
+    token = create_jwt_token(Config.ADMIN_CLIENT_SECRET, client_id=Config.ADMIN_CLIENT_USER_NAME)
+    to = Config.EMAIL_TO if notification_type == "email" else Config.SMS_TO
+    template_id = Config.EMAIL_TEMPLATE_ID if notification_type == "email" else Config.SMS_TEMPLATE_ID
```

Comment:
It's very nice to have this code as a recipe on how to programmatically connect to the admin as a client in a succinct excerpt.